### ISSUE TITLE
Allow events in a readonly Q-Input appended slot

### ISSUE THREAD

**Issue by drewrawitz** [quasarframework/quasar#4767]
**Is your feature request related to a problem? Please describe.**
Here's a screenshot so you can visualize what I'm currently working with:

<img width="553" alt="Screen Shot 2019-07-30 at 4 01 05 PM" src="https://user-images.githubusercontent.com/5014932/62169136-972b5d80-b2e4-11e9-8ffb-c6f4c63ba61c.png">

I'd like to display a tooltip when the help icon is hovered.  Here's the current code:

```
<q-input
  value="myemail@gmail.com"
  type="text"
  outlined
  readonly
>
  <template v-slot:append>
    <q-icon name="help_outline" size="19px" />
    <q-tooltip
      anchor="top middle"
      self="bottom middle"
      :offset="[0, 0]"
      content-class="bg-black"
      :content-style="{ 'font-size': '13px', 'text-align': 'center' }"
    >
      This is the tooltip text
    </q-tooltip>
  </template>
</q-input>
```

But because the input is `readonly`, the tooltip never gets shown.

**Describe the solution you'd like**
Have the actual input be a `readonly` field but still allow events on any of the input slots.

**Describe alternatives you've considered**
Removing the `readonly` prop and creating my own custom functionality to disable just the input. Not ideal, it would be great if this was baked in.

**Comment by olegvakulenko** [comment]
This is by design. Use `before` or `after` slot.

**Comment by rstoenescu** [comment]
Fix will be available in 1.0.6.
